content/resources/faq.md

```diff
@@ -28,6 +33,12 @@ GitBOM seeks to have the following characteristics:
 
 The uniqueness requirement is what drives the exclusion of metadata from the GitBOM.
 
-Metadata out of tree can always reference artifacts by their artifact ID or parent child relationships by the GitBOM id.
+Metadata, which is stored outside of the GitBOM tree, may reference any artifact within the tree by its artifact ID, and can also reference a parent-child relationship in the same way.
+
+## Won't the generation of artifact trees slow down build processes?
+
+We don't think so, and would be delighted to receive data from any very large projects which would either challenge or validate this assumption.
+
+## What about files with duplicate hashes?
 
-### Why isn't information about the compiler/linker included in the GitBOM?
\ No newline at end of file
+Duplicate hashes are very likely to occur in empty and trivially-small files, and files which are copied frequently, such as LICENSE files. We don't think this is a problem because GitBOM is not an SBOM -- it does not provide provenance metadata about any files or artifacts, the domain of SBOM tools and standards such as SPDX and CycloneDX.
```

Comment:
It might be worth separating the distinction between files-of-identical-content which *should* have duplicate hashes and actual collisions between files-with-different-content.  So far GitHub has yet to observe collisions between files that are not content identical... so I'm not sure its true that duplicate hashes are very likely to occur for files-with-different-content at all...

Reply:
Hmm. I meant to indicate that files-with-duplicate-contents are very likely to be members of the following classes of files: empty, trivially small, and files copied frequently such as LICENSE files.

Reply:
To my eye most of what was communicated was that duplicates are very likely to occur...

Reply:
Fair enough. I took another pass at the wording of this FAQ.